include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -407,9 +411,13 @@ class CollectiveEpilogue<
       CUTLASS_PRAGMA_UNROLL
       for (int epi_m = 0; epi_m < FragsM; epi_m++) {
         cst_callbacks.begin_loop(epi_m, epi_n);
-
-        if (is_C_load_needed) {
-          copy(params.xe_load_c, tCgC(_, epi_m, epi_n), trC);
+        
+        //Instead of calling is_C_load_needed. We do heirachical check 
+        //so that runtime check not there when ElementC is void 
```

Comment:
the compiler will likely optimize the runtime check away even without the hierarchical check. This is really only there to avoid the compiler error in instantiating the copy